rgw/v2/tests/s3_swift/configs/test_lc_transition_with_lc_process.yaml

```diff
@@ -0,0 +1,34 @@
+#test_bucket_lifecycle_object_expiration_transition.py
```

Comment:
update both config files in test_bucket_lifecycle_object_expiration_transition.py

Reply:
done